pymeasure/instruments/fluke/fluke7341.py

```diff
@@ -0,0 +1,68 @@
+#
+# This file is part of the PyMeasure package.
+#
+# Copyright (c) 2013-2021 PyMeasure Developers
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+#
+
+from pymeasure.instruments import Instrument
+from pymeasure.instruments.validators import strict_discrete_set, strict_range
+from pymeasure.adapters import SerialAdapter
+
+class FlukeSerialAdapter(SerialAdapter):
+    def write(self, command):
+        super().write(command + "\r\n")
+
+class Fluke7341(Instrument):
+    """ Represents the compact constant temperature bath from Fluke
+    """
+
+    set_point = Instrument.control("s", "s=%g",
+                                   """ Set the bath temperature. This property can be read
+                                   """,
+                                   validator=strict_range,
+                                   values = (-40, 150),
+                                   get_process=lambda x: float(x.split()[1])
+                                   )
+    unit = Instrument.control("u", "u=%s",
+                              """ Set the temperature unit. This property can be read
+                              """,
+                              validator=strict_discrete_set,
+                              values = ('c','f'),
+                              get_process=lambda x: x.split()[1]
+                              )
+    
+    temperature = Instrument.measurement("t",
+                                         """ Read the current bath temperature """,
```

Comment:
I would mention that the unit is the same as the one provided by `unit` to prevent confusion

Reply:
OK, I will update documentation